File: finance-app-backend/src/main/java/com/victor/financeapp/backend/application/dto/MonthClosureDTO.java
package com.victor.financeapp.backend.application.dto;

import java.math.BigDecimal;

public record MonthClosureDTO(
        String month,
        Integer year,
        BigDecimal total,
        BigDecimal available,
        BigDecimal expenses,
        Integer index,
        BigDecimal finalUsdToBRL
) {
}

File: finance-app-android/app/src/main/java/com/viictrp/financeapp/data/remote/dto/MonthClosureDTO.kt
package com.viictrp.financeapp.data.remote.dto

import java.math.BigDecimal

data class MonthClosureDTO(
    val month: String,
    val year: Int,
    val total: BigDecimal,
    val available: BigDecimal,
    val expenses: BigDecimal,
    val index: Int,
    val finalUsdToBRL: BigDecimal
)
